src/osgEarth/GEOS.cpp

```diff
@@ -63,9 +63,13 @@ namespace
         {
             coords->push_back( coords->front() );
         }
+#if GEOS_VERSION_AT_LEAST(3,8)
+        geom::CoordinateSequence::Ptr seq = factory->create(coords);
+        return seq.get();
```

Comment:
This seems wrong. Ptr is a unique_ptr so the memory will be freed up as soon as the function is returned.
This function  would be returning freed up memory.

seq.release is probably what would work here but there migh be a memory leak